### DISCUSSION TITLE
Play Books Demo

### DISCUSSION THREAD

**Discussion by usmanaa10** [Security-Onion-Solutions/securityonion#7767] [Show and tell] [answered]
Hi friends,
             I want to know can anyone help me in how to use playbooks and how can it generate alert any demo or steps or snaps shot that can help me for POC..
I have make all available playbooks active but didn't get any kind of alert yet ..how can i generate the dummy alert or something like that.
Any steps and configuration that can help me ..

Thanks

**Comment by defensivedepth** [comment, marked as answer]
Please review our free Essentials training: https://www.youtube.com/watch?v=IS2SOlDedPc&list=PLljFlTO9rB155aYBjHw2InKkSMLuhWpxH&index=7

**Reply by usmanaa10** [reply]
Thanks

**Reply by usmanaa10** [reply]
I follow all steps.
I created sign-in Play and make it active
Then I saw event ID in hunt and found the logs but didn't saw any alert by playbook in Alert Dashboard. Why?
Anyone know what I did wrong

**Reply by dougburks** [reply]
Are you able to share screenshots of what you did?

**Reply by usmanaa10** [reply]
I dont know but when i cam next day and saw the alerts i got Plyabook alerts
